features/bsm/vaadin-adminpage/src/main/java/org/opennms/netmgt/bsm/vaadin/adminpage/BusinessServiceEditWindow.java

```diff
@@ -202,9 +217,16 @@ public void buttonClick(Button.ClickEvent event) {
          */
         m_nameTextField = new TextField("Business Service Name");
         m_nameTextField.setId("nameField");
-        m_nameTextField.setValue(businessService.getName());
+        m_nameTextField.setValue(Strings.nullToEmpty(businessService.getName()));
```

Comment:
Just a note: Vaadin can deal with this.
There is a `setNullRepresentation` and `setNullValue` method for this.